zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/SearchQueryRequestMapper.java

```diff
@@ -240,6 +261,29 @@ private static UserTaskFilter toUserTaskFilter(final UserTaskFilterRequest filte
     return builder.build();
   }
 
+  private static UserFilter toUserFilter(final CamundaUserFilterRequest filter) {
+    final var builder = FilterBuilders.user();
+
+    if (filter != null) {
+      // username
+      if (filter.getUsername() != null) {
```

Comment:
🔧  Minor suggestion, we can use Optionals here in order to refactor  
(There are some endpoints that are using already)
I am intending to refactor this later.